ext/uri/config.w32

```diff
@@ -1,9 +1,9 @@
-EXTENSION("uri", "php_lexbor.c php_uri.c php_uri_common.c php_uriparser.c", false /* never shared */, "/I ext/lexbor /I ext/uri/uriparser/include /DZEND_ENABLE_STATIC_TSRMLS_CACHE=1");
+EXTENSION("uri", php_uri.c php_uri_common.c "uri_parser_rfc3986.c uri_parser_whatwg.c ", false /* never shared */, "/I ext/lexbor /I ext/uri/uriparser/include /DZEND_ENABLE_STATIC_TSRMLS_CACHE=1");
```

Comment:
```suggestion
EXTENSION("uri", php_uri.c php_uri_common.c uri_parser_rfc3986.c uri_parser_whatwg.c", false /* never shared */, "/I ext/lexbor /I ext/uri/uriparser/include /DZEND_ENABLE_STATIC_TSRMLS_CACHE=1");
```

Reply:
Windows build fails, and it's related to this line, the opening `"` is misplaced (and gone in Tim's suggestion).